server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -1402,10 +1402,8 @@ public void loadTablet(TInfo tinfo, TCredentials credentials, String lock,
       synchronized (server.openingTablets) {
         synchronized (server.onlineTablets) {
 
-          // checking if this exact tablet is in any of the sets
-          // below is not a strong enough check
-          // when splits and fix splits occurring
-
+          // With splits occurring, checking if the current tablet
+          // is in any of the sets below may not be a strong enough check
```

Comment:
```suggestion
          // Checking if the current tablet is in any of the sets 
          // below is not a strong enough check to catch all overlapping tablets
          // when splits and fix splits are occurring
```

With the changed to the error message, I think the original comment along with the small change here is enough of an improvement.